resources/tenants/controller/tenants.go

```diff
@@ -54,3 +66,23 @@ func (c *Controller) ProvisionTenantsHandler(w rest.ResponseWriter, r *rest.Requ
 
 	w.WriteHeader(http.StatusCreated)
 }
+
+func (c *DeploymentsControllerWithTenant) DeploymentsPerTenantHandler(w rest.ResponseWriter, r *rest.Request) {
+	ctx := r.Context()
+	l := log.FromContext(ctx)
+	defer r.Body.Close()
+
+	tenantID := r.PathParam("tenant")
+
+	if tenantID == "" {
+		rest_utils.RestErrWithLog(w, r, l, nil, http.StatusBadRequest)
+	}
+
+	//REVIEW: WithoutAuthContext would need to be added to go-lib-micro
+	tenantCtx := identity.WithoutAuthContext(ctx, &identity.Identity{
+		Tenant: tenantID,
+	})
+
+	r.Request = r.WithContext(tenantCtx)
+	c.deploymentsController.LookupDeployment(w, r)
```

Comment:
share model instead model (once business logic is cleaned up and moved down the stack)

Reply:
but the controller is already doing all the query parsing, I'd have to re-implement it here .